README.md

```diff
@@ -548,21 +551,27 @@ This repository contains a curated list of awesome open source libraries that wi
 * [Diffgram](https://diffgram.com/) - Training Data First platform. Database & Training Data Pipelines for Supervised AI. Integrated with GCP, AWS, Azure and top Annotation Supervision UIs (or use built-in Diffgram UI, or build your own). Plus a growing list of integrated service providers! For Computer Vision, NLP, and Supervised Deep Learning / Machine Learning.
 * [Domino](https://www.dominodatalab.com/) - An enterprise MLOps platform that supports data scientist collaboration with their preferred tools, languages, and infrastructure, with IT central resource management, governance, and security, without vendor lock-in.
 * [Fennel](https://fennel.ai/) - Realtime feature engineering platform for fast moving machine learning teams. Python / Pandas native, built in Rust. Easy to install/use/run, builds upon best practices for reducing data/feature quality issues, and keeps cloud spend low. Fully managed, zero ops.
+* [Fiddler](https://www.fiddler.ai/) - Fiddler is a model performance management platform that offers model monitoring, observability, explainability & fairness.
 * [Google Cloud Machine Learning Engine](https://cloud.google.com/ml-engine/) - Managed service that enables developers and data scientists to build and bring machine learning models to production.
 * [Graphsignal](https://graphsignal.com/) - Machine learning profiler that helps make model training and inference faster and more efficient.
 * [H2O Driverless AI](https://www.h2o.ai/products/h2o-driverless-ai/) - Automates key machine learning tasks, delivering automatic feature engineering, model validation, model tuning, model selection and deployment, machine learning interpretability, bring your own recipe, time-series and automatic pipeline generation for model scoring - [(Video)](https://www.youtube.com/watch?v=ZqCoFp3-rGc).
+* [Hugging Face](https://huggingface.co/) - Hugging Face is a platform that allows users to share machine learning models and datasets.
```

Comment:
HuggingFace is one word

Reply:
I initially thought so, but it is actually two words. @axsaucedo 
![image](https://user-images.githubusercontent.com/8592144/219959145-124d3609-8d5e-4588-8bf0-3008b1a2fde5.png)